src/annotator/integrations/pdf.js

```diff
@@ -241,13 +262,26 @@ export class PDFIntegration {
       return;
     }
 
-    this._warningBanner = document.createElement('hypothesis-banner');
-    document.body.prepend(this._warningBanner);
+    if (!this._banner) {
+      this._banner = document.createElement('hypothesis-banner');
+      document.body.prepend(this._banner);
+      createShadowRoot(this._banner);
+    }
 
-    const warningBannerContent = createShadowRoot(this._warningBanner);
-    render(<WarningBanner />, warningBannerContent);
+    render(
+      <Banners>
```

Comment:
I pushed _just one more commit_ with a dead simple `Banners` component. Explanation:

* Tailwind only includes utility CSS classes in output that are used in any of the modules/files in its defined `content` globs
* I'd like to avoid expanding the globs to include non-component source (for cleanliness and performance) except where necessary—this opinion is certainly open for discussion!
* These classes "worked" here because `flex flex-col` are commonly used and have been used in other components in the application (and thus were included in output styles)

The `Banners` component is dead simple and is exercised by this module's tests (i.e. I don't think that display-only component needs "real" tests).

Reply:
> Tailwind only includes utility CSS classes in output that are used in any of the modules/files in its defined content globs

Ah - I forgot about that. Thanks for the fixup.

> The Banners component is dead simple and is exercised by this module's tests (i.e. I don't think that display-only component needs "real" tests).

I agree. Such a trivial component just needs _some_ test to render it.